### DISCUSSION TITLE
How is a new sensor session started while an old one is active now that the Stop Sensor option has been removed?

### DISCUSSION THREAD

**Discussion by asavageiv** [NightscoutFoundation/xDrip#3401] [Q&A] [answered]
Do I just change the transmitter ID and then go to the status page? I don't see "Start Sensor" after I change the ID. I did that while an old sensor session was still running (had about 15 minutes left) and so far xDrip+ is giving me `Sensor Status: Mismatch - wait` on the Status screen. I've confirmed several times the new ID is correct.

Related to https://github.com/NightscoutFoundation/xDrip/discussions/3400

**Comment by Navid200** [comment, marked as answer]
What xDrip release are you using?
You can see on the classic status page.

A G7 session starts on G7 as soon as you use the applicator.  There is no need to do anything else to start it.
xDrip can connect to a session in progress by entering the corresponding transmitter ID.

**Reply by asavageiv** [reply]
![Screenshot_20240324-112855](https://github.com/NightscoutFoundation/xDrip/assets/3028205/bdd8f9e8-2e88-4a93-94be-f5e2ee1e92f5)


Plus some extra info
![Screenshot_20240324-113753](https://github.com/NightscoutFoundation/xDrip/assets/3028205/ec535f8b-edda-4cc0-b0e7-80897f7b475b)
![Screenshot_20240324-112925](https://github.com/NightscoutFoundation/xDrip/assets/3028205/4e97bbff-b500-46ee-b1c2-709b44fbe665)
![Screenshot_20240324-112902](https://github.com/NightscoutFoundation/xDrip/assets/3028205/865c0d79-3c65-46a2-8637-a477f812e0df)

**Reply by Navid200** [reply]
There is a log that says sensor stopped at 10:46.
What caused that?
Do you have a Dexcom receiver?  Do you have a pump?
Is there any other device or app that is connected to the same G7?

**Comment by asavageiv** [comment]
Ah, I forgot to mention, after it had failed to connect for quite a while I
enabled engineering mode, changed the transmitter ID and used "stop", then
changed the ID back. I've never hit the pairing request for the new sensor,
so I doubt the "stop" did anything to the new one.

 I don't have any other receiver connected to it, but AAPS gets data from
xDrip+ on my phone.

**Reply by Navid200** [reply]
Have you inserted the new sensor?
Do you have used G7 devices nearby?
Tap on restart collector.
Raise the volume so that you can hear the pink.

**Comment by asavageiv** [comment]
Yeah, I inserted the new sensor about 30 minutes before changing the
transmitter ID the first time. The old G7s are quite far away now (in
another county). Volume is up and the app has been on that page for >25
minutes at a time. Should I be seeing pairing attempts in the event log?

**Reply by Navid200** [reply]
It will eventually connect.
If you have Nightscout and have all your readings already uploaded, you can uninstall xDrip, reinstall, choose G7 in source wizard, tap on settings to bring up the request to give permission, choose to do so in settings and select all the time, scan the QR code,  change battery optimization to unrestricted, enter the transmitter ID and wait for pair request.

You can also backup your database before uninstalling.

**Reply by asavageiv** [reply]
No luck even after uninstall and reinstall. I ended up trying with the G7 app and that didn't work either. I gave up on that sensor and put in a new one and that one works fine. Must have been a bad sensor.
